ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -230,6 +239,9 @@ - (void)requestCode:(ADAuthorizationCodeCallback)completionBlock
                                                                         errorDetails:@"The authorization server did not return a valid authorization code."
                                                                        correlationId:[_requestParams correlationId]];
                      }
+                     
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)